Helm/opensearch-dashboards/templates/deployment.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "opensearch-dashboards.fullname" . }}
+  labels: {{ include "opensearch-dashboards.labels" . | nindent 4 }}
```

Comment:
This should use `{{- ... }}` to avoid a trailing space on the line after `label: `. This is true for any value using `nindent`.

Reply:
Will change it to `replicaCount`.